pkgs/tools/misc/uudeview/default.nix

```diff
@@ -1,18 +1,35 @@
-{ stdenv, fetchurl, tcl, tk }:
+{ stdenv
+, fetchurl
+, fetchpatch
+, tcl
+, tk
+}:
 
 stdenv.mkDerivation rec {
-  name = "uudeview-0.5.20";
+  pname = "uudeview";
+  version = "0.5.20";
+
   src = fetchurl {
-    url = "http://www.fpx.de/fp/Software/UUDeview/download/${name}.tar.gz";
+    url = "http://www.fpx.de/fp/Software/UUDeview/download/${pname}-${version}.tar.gz";
     sha256 = "0dg4v888fxhmf51vxq1z1gd57fslsidn15jf42pj4817vw6m36p4";
   };
 
   buildInputs = [ tcl tk ];
-  hardeningDisable = [ "format" ];
+
   configureFlags = [ "--enable-tk=${tk.dev}" "--enable-tcl=${tcl}" ];
 
-  # https://wiki.tcl.tk/3577
-  patches = [ ./matherr.patch ];
+  patches = [
+    # https://wiki.tcl.tk/3577
+    ./matherr.patch
```

Comment:
Why is this patch required? Renaming can be done with a command.

Reply:
that patch was introduced in 14fc982e0c03e3ef220a95a0f724b02d4c916a8f by @Infinisil and I do not touch it. I only modify the patch header to be compatible to patchFlags = [ "-p0" ] which I must set for the newly added patch